ext/src/ruby_api/convert.rs

```diff
@@ -45,25 +55,31 @@ impl ToWasmVal for Value {
             ValType::ExternRef => {
                 let extern_ref_value = match self.is_nil() {
                     true => None,
-                    false => Some(ExternRef::new(OnStackValue::from(*self))),
+                    false => Some(ExternRef::new(RbExternRef::from(*self))),
                 };
 
                 Ok(Val::ExternRef(extern_ref_value))
             }
-            ValType::FuncRef => Ok(Val::FuncRef(Some(*self.try_convert::<&Func>()?.inner()))),
+            ValType::FuncRef => {
+                let func_ref_value = match self.is_nil() {
+                    true => None,
+                    false => Some(*self.try_convert::<&Func>()?.inner()),
+                };
+                Ok(Val::FuncRef(func_ref_value))
+            }
             ValType::V128 => err!("converting from Ruby to v128 not supported"),
         }
     }
 }
 
-struct OnStackValue(Value);
-impl From<Value> for OnStackValue {
+struct RbExternRef(Value);
```

Comment:
Is there any advantage on prefixing this with `Rb`? `ExternRefValue(Value)` maybe, since `Value` is what we really work with? I don't think we have adopted the `Rb*` naming elsewhere, aside from whatever we consume from Magnus (if anything).

Reply:
You are right that we do not use this naming style elsewhere. Will update.